pybossa/api/task.py

```diff
@@ -114,3 +120,22 @@ def _select_attributes(self, data):
             data.pop('gold_answers', None)
             data.pop('calibration', None)
         return data
+
+    def upload_gold_data(self, task, project_id, private_gold_answers, file_id=None):
```

Comment:
Is this used anywhere else?

Reply:
not now but I was thinking that it might get used during the refactor on the next ticket.